docker-compose.yml

```diff
@@ -99,7 +99,7 @@ services:
       - es
       - indexer-worker
     volumes:
-      - ./ingestion_server:/ingestion-server
+      - ./ingestion_server:/ingestion_server
```

Comment:
I think the same change needs to be done on line 132 (`ingestion-server` -> `ingestion_server`)

Reply:
This PR entirely left out any changes to the indexer workers. I was considering to do that in a separate PR.

Reply:
Addressed in #138.